lib/include/hse/ikvdb/throttle.h

```diff
@@ -6,6 +6,59 @@
 #ifndef HSE_KVDB_THROTTLE_H
 #define HSE_KVDB_THROTTLE_H
 
+/**
```

Comment:
I might move this underneath all the includes, but up to you

Reply:
I think it's less disruptive at the top of the file rather than being interleaved with the code..